src/facade/dragonfly_listener.cc

```diff
@@ -356,6 +365,7 @@ void Listener::OnConnectionClose(util::Connection* conn) {
 }
 
 void Listener::OnMaxConnectionsReached(util::FiberSocketBase* sock) {
+  listener_tl_stats.maxclients_reached_cnt++;
```

Comment:
This will be called for all listeners, right? (i.e. not only TLS listeners). As such, why do you call the function that returns it `TLSMaxClientsReachedCount()` and not `MaxClientsReachedCount()`?